edx_proctoring/models.py

```diff
@@ -394,27 +394,20 @@ def get_exam_attempt_by_code(self, attempt_code):
             exam_attempt_obj = None
         return exam_attempt_obj
 
-    def get_all_exam_attempts(self, course_id, timed_exams_only=False):
+    def get_all_exam_attempts(self, course_id):
         """
         Returns the Student Exam Attempts for the given course_id.
         """
         filtered_query = Q(proctored_exam__course_id=course_id)
-
-        if timed_exams_only:
-            filtered_query = filtered_query & Q(proctored_exam__is_proctored=False)
-
         return self.filter(filtered_query).order_by('-created')
 
-    def get_filtered_exam_attempts(self, course_id, search_by, timed_exams_only=False):
+    def get_filtered_exam_attempts(self, course_id, search_by):
```

Comment:
you are removing `timed_exams_only` kwarg, have you checked it will be backward compatible?
I still see this kw argument being used in https://github.com/edx/edx-proctoring/blob/86ded2133afe7c9df70569c1f639b4392dd12db9/edx_proctoring/api.py#L356-L363 
Are you aware of this?

Reply:
oh i was really not aware of it. Thank you for highlighting 👍. I am going to test it out.